.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,54 @@
+name: Integration tests
+on:
+  pull_request:
+env:
+  TAG: pr-${{ github.event.number }}
+  # VERSION string has to begin with x.y.z format, required by bundle and catalog build commands
+  VERSION: 2.26.0-pr-${{ github.event.number }}
+  IMAGE_TAG_BASE: quay.io/<username-or-org>/opendatahub-operator
+jobs:
+  create-catalog-image:
+    name: Build and push catalog image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_BUILDER: docker
+    steps:
+      - name: Quay.io login
+        uses: docker/login-action@v3
```

Comment:
Maybe use something in-house 😉  https://github.com/redhat-actions/podman-login

Reply:
Thanks Ajay, I will keep this in mind and include podman action later once I move on from local testing